### DISCUSSION TITLE
Roadmap for lightweight_mmm

### DISCUSSION THREAD

**Discussion by exolab** [google/lightweight_mmm#175] [General]
I wonder if there is a public roadmap for this project. 

The release notes for v0.1.7 mention `Internal changes in preparation for a larger update.` Can anyone shed some light on what this large update will bring?

**Comment by steven-struglia** [comment]
Agreed, would love to know what's coming down the pipeline for this project.

**Comment by AruunnAnanda** [comment]
Even I would be interested in knowing it
